react-form-controls/src/index.jsx

```diff
@@ -0,0 +1,36 @@
+import React, { useState } from 'react';
+import ReactDOM from 'react-dom/client';
+
+function RegistrationForm() {
+  const [changeUser, setChangeUser] = useState('');
+  const [changePassword, setChangePassword] = useState('');
+
+  function handleUserNameChange(e) {
+    setChangeUser(e.target.value);
+  }
+  function handlePasswordChange(eve) {
+    setChangePassword(eve.target.value);
+  }
+  function handleSubmit(eon) {
+    eon.preventDefault();
+    const form = eon.target;
```

Comment:
Alternatively, since you have the state in the `changeUser` and `changePassword` variables, you could just log those.